### PR TITLE
Remove torrent data support

### PR DESCRIPTION
**Pull request by mathiasvr**
Support for moving torrent data to trash, and context menu items as proposed in #560.

### CODE REVIEW

**Comment by dcposch** [comment]
This doesn't have the confirmation dialogs, but I think that's fine, we can add those later. LGTM

File: renderer/main.js [hunk 1]
```diff
@@ -1124,6 +1129,20 @@ function openTorrentContextMenu (infoHash) {
   var torrentSummary = getTorrentSummary(infoHash)
   var menu = new electron.remote.Menu()
 
+  menu.append(new electron.remote.MenuItem({
```
Comment (dcposch) [inline renderer/main.js]: Nitpick, I would probably put these two menu items at the bottom and keep Show In Finder on top

**Comment by dcposch** [comment]
Thanks!

**Comment by feross** [comment]
Just getting a chance to review this now. Excellent PR! 👍

**Pull request merged**